doc/Makefile

```diff
@@ -42,8 +45,13 @@ man-chapel: FORCE
 	../util/config/run-in-venv.bash $(MAKE) man
 
 
-source: module-docs primers hellos symlinks
+source: collect-syntax module-docs primers hellos symlinks
+
 
+collect-syntax: clean-collect-syntax
+	@echo
+	@echo "Collecting syntax from '$(SPECSRCDIR)' into '$(SPECSRCDIR)/syntax.rst'"
+	$(GETSYNTAX) rst/language/spec --outdir $(SPECSRCDIR)
```

Comment:
since it is just run once, why have a GETSYNTAX variable? AFAIK this should just be ./util/collect-syntax.py